lib/OpenQA/Utils.pm

```diff
@@ -166,6 +168,23 @@ sub testcasedir {
     return $dir;
 }
 
+sub gitrepodir {
+    my %args = (distri => '', version => '', repo => 'tests', @_,);
+    my $path
+      = $args{needles}
+      ? needledir($args{distri}, $args{version})
+      : testcasedir($args{distri}, $args{version});
+    my $filename = (-e path($path, '.git')) ? path($path, '.git', 'config') : '';
+    log_warning("$path is not a git directory") if $filename eq '';
+    my $config = Config::Tiny->read($filename, 'utf8');
+    return '' unless defined $config;
+    return $config->{'remote "origin"'}{url} if ($config->{'remote "origin"'}{url} =~ /^http(s?)/);
+    my @url_tokenized = split(':', $config->{'remote "origin"'}{url});
+    $url_tokenized[1] =~ s{\.git$}{/commit/};
+    my @githost = split('@', $url_tokenized[0]);
+    return "https://$githost[1]/${url_tokenized}[1]";
```

Comment:
CI tests fail on this `${url_tokenized}[1]`, don't know if this is correct syntax

Reply:
fixed